src/Libraries/Microsoft.Extensions.Resilience/Resilience/Internal/ResilienceTagNames.cs

```diff
@@ -5,13 +5,13 @@ namespace Microsoft.Extensions.Resilience.Internal;
 
 internal static class ResilienceTagNames
 {
-    public const string FailureSource = "failure-source";
+    public const string FailureSource = "resilience.failure.source";
```

Comment:
`resilience` is a bit vague term and can be used for many different things beyond `Microsoft.Extensions.Resilience`. Can we come up with a more specific prefix? E.g. `ms.resilience` or `aspnetcore.resilience` ?

Reply:
Switched to `dotnet.resilience.*`